itest/lnd_route_blinding_test.go

```diff
@@ -1018,10 +1018,14 @@ func testErrorHandlingOnChainFailure(ht *lntest.HarnessTest) {
 	// suspended Carol we don't need to account for her commitment output
 	// claim.
 	ht.Miner.MineBlocksAndAssertNumTxes(1, 1)
```

Comment:
nit: we could instead call `ht.MineBlocksAndAssertNumTxes`, in which it will assert the nodes are synced to the latest chain.